### ISSUE TITLE
Refactor localization

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#180]
Some thoughts on improving localization:
- Each language should have a folder containing files for each path in the application, as well as certain other aspects like the WCAG translations. This will make it easier to maintain multiple translations. 
- The translation files should be kept in the app folder, and a concatenation should be put in .temp so both aren't stored in GIT
- The user should be able to switch language on the fly. They should be kept separate and only be loaded when they are required.
- Grunt should test if each supported language has the same properties as the default English version does before building

**Comment by WilcoFiers** [comment]
Bullet 1 and 2 are done. Now to actually build in multiple languages.

**Comment by WilcoFiers** [comment]
Finally, done.
